src/Simulation/Simulators/QuantumProcessor/ClassicalControl.cs

```diff
@@ -83,7 +83,7 @@ private static void RunClause(ICallable op, OperationFunctor type, IQArray<Qubit
         }
```

Comment:
In ClassicalControl.qs, I saw that you changed the namespace back to `Microsoft.Quantum.Simulation.QuantumProcessor.Extensions`. Did it break things to use `Microsoft.Quantum.ClassicalControl` instead?

Reply:
Yah, that change requires cross-repo changes. This is the namespace change I brought up in the daily meeting, to which Bettina had suggested I make this a separate task. Once this PR is completed, the functionality will be provided and we can better address what the name should be changed to, and how to go about doing it.

Reply:
That makes sense, maybe there can be a TODO at the top of ClassicalControl.qs mentioning this? (Since the namespace looks inconsistent from the other files right now.)

Reply:
done